### PR TITLE
added a copy button to console log window

### PR DESCRIPTION
**Pull request by jevenski**
Added a copy button to console window, making copying console log like mediainfo result a little bit more confortable, without having to select the entire text and hit Ctrl+C. Log a message and nothing else when the console log is empty, e.g. when screenshots are being generated.

console_log

PS: Apologies again for all the mess by creating pull requests again and again... I made some adjustments with my fork and closed the old request accidentally. Now I'm getting a hang of the git philosophy.

### CODE REVIEW

File: plugins/_task/init.js [hunk 1]
```diff
@@ -116,10 +116,23 @@ plugin.fromBackground = function( no )
...
+  if(consoleLog !== "")
+  {
+    navigator.clipboard.writeText(consoleLog);
+  }
+  else
+  {
+    log("Console log is empty.");
+  }
+}
+
```
Comment (stickz) [inline plugins/_task/init.js:131]: Can you adjust the spacing to match the other items in the Javascript file?

File: plugins/_task/init.js [hunk 2]
```diff
@@ -116,10 +116,23 @@ plugin.fromBackground = function( no )
...
 plugin.toBackground = function()
 {
-       	if(!plugin.isInBackground())
-		plugin.background[ plugin.foreground.no ] = cloneObject( this.foreground );
+  if(!plugin.isInBackground())
+	plugin.background[ plugin.foreground.no ] = cloneObject( this.foreground );
```
Comment (stickz) [inline plugins/_task/init.js:135]: Formatting.

File: plugins/_task/init.js [hunk 3]
```diff
@@ -594,6 +607,7 @@ plugin.onLangLoaded = function()
 			"</fieldset>"+
 		"</div>"+
 		"<div class='aright buttons-list' id='tsk_btns'>"+
+      "<input type='button' id='tskCopy' class='Button' value='"+theUILang.tskCopy+"'/>"+
```
Comment (stickz) [inline plugins/_task/init.js:610]: Formatting.

File: plugins/_task/init.js [hunk 4]
```diff
@@ -614,4 +628,5 @@ plugin.onLangLoaded = function()
 	});
 	$('#tskBackground').on('click', plugin.toBackground );
 	$(".tskconsole").enableSysMenu();
+  $("#tskCopy").on('click', plugin.copyConsoleLog);
```
Comment (stickz) [inline plugins/_task/init.js:631]: Formatting.

**Review by stickz — CHANGES_REQUESTED** [review]
Looks good, just need to adjust spacing and resolve review.

**Comment by jevenski** [comment]
Submitted another commit regarding the indent format changes.

**Comment by koblack** [comment]
Copy from mediainfo and unpack plugins throw an error
```
webui.js?v=430:2482 Uncaught TypeError: Cannot read properties of undefined (reading 'writeText') from TypeError: Cannot read properties of undefined (reading 'writeText')
    at plugin.copyConsoleLog (<anonymous>:131:23)
    at HTMLInputElement.dispatch (http://xxx/ruTorrent-develop/js/jquery.js?v=430:2:40035)
    at v.handle (http://xxx/ruTorrent-develop/js/jquery.js?v=430:2:38006)
```

**Pull request merged**